guardrails/validators.py

```diff
@@ -2229,3 +2229,140 @@ def query_vector_collection(
             raise ValueError("distance_metric must be 'cosine'.")
 
         return top_chunks
+
+
+@register_validator(name="distribution-check", data_type="string")
+class SimilarToList(Validator):
+    """Validates that a value is similar to a list of previously known values.
```

Comment:
add a callout that no fix value is available.

Reply:
Is callout some property to add somewhere or you mean just to add text in the doc string?
cc @CalebCourier

Reply:
That would be in the doc string. See here: https://github.com/ShreyaR/guardrails/blob/main/guardrails/validators.py#L448